README.md

```diff
@@ -186,11 +189,11 @@ sudo -u postgres psql -d lisk_test -c "alter user lisk with password 'password';
 sudo -u postgres psql -d lisk_main -c "alter user lisk with password 'password';"
 ```
 
-* MacOS 10.12-10.13 (Sierra/High Sierra):
+* MacOS 10.12-10.14 (Sierra/High Sierra/Mojave)::
 
 ```
 brew install postgresql@10
-initdb /usr/local/var/postgres --encoding utf8 --locale=en_US.UTF-8
+initdb /usr/local/var/postgres@10 --encoding utf8 --locale=en_US.UTF-8
```

Comment:
I'm not sure if there is `@10` at the folder name. Could you please double check.

Reply:
Yes, that issue I came across when installing for Mac.
@SargeKhan can confirm.